Makefile

```diff
@@ -1,39 +1,70 @@
+# Use /bin/bash instead of /bin/sh
+export SHELL = /bin/bash
+
 ## ========================================
 ## Commands for both workshop and lesson websites.
 
 # Settings
 MAKEFILES=Makefile $(wildcard *.mk)
-JEKYLL=jekyll
-JEKYLL_VERSION=3.7.3
+JEKYLL_VERSION=3.8.5
+JEKYLL=bundle install --path .vendor/bundle && bundle update && bundle exec jekyll
```

Comment:
If I understand correctly, you are updating dependencies on each page build. I guess that is why you deleted the Gemfile.lock as well? Is this common practice for the Carpentries?